code/game/atoms_movable.dm

```diff
@@ -225,6 +225,17 @@
 		A.CollidedWith(src)
 
 /atom/movable/proc/forceMove(atom/destination)
+	. = FALSE
+	if(destination)
+		. = doMove(destination)
+	else
+		CRASH("No valid destination passed into forceMove")
+
+/atom/movable/proc/moveToNullspace()
+	return doMove(null)
+
+/atom/movable/proc/doMove(atom/destination)
+	. = FALSE

```

Comment:
Isn't it impossible for the result to be FALSE now? (As long as the proc doesn't CRASH

Reply:
this is in case it does